libfreerdp/codec/include/bitmap.c

```diff
@@ -130,8 +145,11 @@ static INLINE BOOL RLEDECOMPRESS(const BYTE* pbSrcBuffer, UINT32 cbSrcBuffer, BY
 		*/
 		code = ExtractCodeId(*pbSrc);
 
+		WLog_VRB(TAG, "[%s] pbSrc=%p code=%s, rem=%" PRIuz, __FUNCTION__, pbSrc,
```

Comment:
I am not sure we want to have this inside the loop of a decoding function. This will be very very noisy if using verbose logging while offering not a lot of useful information and it will waste quite a couple of cycles even in release builds on every iteration.

Reply:
@mfleisz the `WINPR_ASSERT` stuff shouldn´t matter, as we will deactivate the default once the `master` branch stabilizes. (then these checks are no longer compiled in). (so, anyone interested can already deactivate it manually, but this way we get feedback from the lazy ones :grin: )
as for the log message, true, I´ll add a define like the other extended debug stuff optionally compiled.

Reply:
@mfleisz ok, added `WITH_DEBUG_CODECS` and deactivated all non `ERROR`, `INFO` or `WARN` messages in interleaved/progressive/...